### PR TITLE
fixes panic in openshift-test cmd

### PR DESCRIPTION
**Pull request by p0lyn0mial**
This PR fixes the following panic when running `openshift-tests run openshift/conformance/serial --dry-run `

```                                                                                                                                                                  
goroutine 1 [running, locked to thread]:                                                                                                                                                                                                                                                                            
runtime/debug.Stack(0x6d, 0xc0038c58c0, 0x2a0)                                                                                                                                                                                                                                                                      
        /usr/local/go/src/runtime/debug/stack.go:24 +0x9d                                                                                                                                                                                                                                                           
github.com/openshift/origin/test/extended/util.FatalErr(0x567ff20, 0xc0039910e0)                                                                                                                                                                                                                                    
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/client.go:643 +0x26                                                                                                                                                                  
github.com/openshift/origin/test/extended/util.(*CLI).AdminConfig(0xc0039bf000, 0xa)                                                                                                                                                                                                                                
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/client.go:443 +0x65                                                                                                                                                                  
github.com/openshift/origin/test/extended/cli.glob..func4()                                                                                                                                                                                                                                                         
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cli/explain.go:332 +0x9e                                                                                                                                                                  
github.com/openshift/origin/vendor/github.com/onsi/ginkgo/internal/suite.(*Suite).PushContainerNode(0xc0002914f0, 0x5f44669, 0x10, 0x61d5558, 0x0, 0x9918b62, 0x80, 0x148, 0xc0039e33b0, 0x8c)                                                                                                                      
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/github.com/onsi/ginkgo/internal/suite/suite.go:144 +0x158                                                                                                                                        
github.com/openshift/origin/vendor/github.com/onsi/ginkgo.Describe(0x5f44669, 0x10, 0x61d5558, 0xc0038a8d01)                                                                                                                                                                                                        
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/github.com/onsi/ginkgo/ginkgo_dsl.go:293 +0xaa                                                                                                                                                   
github.com/openshift/origin/test/extended/cli.init()                                                                                                                                                                                                                                                                
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cli/explain.go:328 +0xb5                                                                                                                                                                  
                                                                                                                                                                                                                                                                                                                    
Jan 22 09:43:29.262: FAIL: open : no such file or directory                                                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                                                                                    
Full Stack Trace                                                                                                                                                                                                                                                                                                    
github.com/openshift/origin/test/extended/util.FatalErr(0x567ff20, 0xc0039910e0)                                                                                                                                                                                                                                    
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/client.go:644 +0x118                                                                                                                                                                 
github.com/openshift/origin/test/extended/util.(*CLI).AdminConfig(0xc0039bf000, 0xa)                                                                                                                                                                                                                                
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/client.go:443 +0x65                                                                                                                                                                  
github.com/openshift/origin/test/extended/cli.glob..func4()                                                                                                                                                                                                                                                         
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cli/explain.go:332 +0x9e                                                                                                                                                                  
github.com/openshift/origin/test/extended/cli.init()                                                                                                                                                                                                                                                                
        /Users/lszaszki/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cli/explain.go:328 +0xb5                                                                                                                                                                  
Jan 22 09:43:30.652: INFO: Could not get network operator configuration: not adding any plugin-specific skips                                                                                                                                                                                                       
W0122 09:43:30.770256   88003 test_context.go:410] Unable to find in-cluster config, using default host : http://127.0.0.1:8080            
```

### CODE REVIEW

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -329,7 +329,6 @@ var _ = g.Describe("[cli] oc explain", func() {
 	defer g.GinkgoRecover()
 
 	oc := exutil.NewCLI("oc-explain", exutil.KubeConfigPath())
-	crdClient := apiextensionsclientset.NewForConfigOrDie(oc.AdminConfig())
```
Comment (p0lyn0mial) [inline test/extended/cli/explain.go]: `oc.AdminConfig` loads a config file from a disk - this caused panic when the config file wasn't present. Having a config is not always required for example when passing `dry-run` flag.

I glanced over `test/extended` just to see if the other tests don't load `AdminConfig` too early.

**Comment by smarterclayton** [comment]
/hold

This is fixed in another LGTM'd PR

**Comment by p0lyn0mial** [comment]
closing as it was fixed in https://github.com/openshift/origin/pull/24445

**Pull request closed without merging**